src/tribler-core/tribler_core/modules/torrent_checker/torrent_checker.py

```diff
@@ -225,7 +244,7 @@ def update_torrents_checked(self, new_result):
         new_result_tuple = (new_result['infohash'], new_result['seeders'],
                             new_result['leechers'], new_result['last_check'])
         if new_result['seeders'] > 0:
-            self.torrents_checked.add(new_result_tuple)
+            self._torrents_checked[new_result['infohash']] = (new_result_tuple)
```

Comment:
```suggestion
            self._torrents_checked[new_result['infohash']] = new_result_tuple
```

Probably, parentheses are redundant here.

Reply:
NIT
`new_result['infohash']` are used twice. So, we can extract it to
```
infohash = new_result['infohash']
```